packages/preview2-shim/lib/nodejs/http.js

```diff
@@ -179,32 +179,59 @@ class RequestOptions {
   #connectTimeout;
   #firstByteTimeout;
   #betweenBytesTimeout;
+  #defaultTimeoutMs = 600_000n;
+  #nanoToMilliConversion = 1_000_000n;
```

Comment:
These don't need to be a private field, instead they can just be top-level constants like `const defaultTimeoutMs = ...; const nanoTomilliConversion = ...`.

Reply:
Updated in https://github.com/bytecodealliance/jco/pull/404/commits/dcf0f31934b9b81b560a7190bc8c0fae8608223f